.github/workflows/images-legacy.yaml

```diff
@@ -171,7 +171,7 @@ jobs:
         uses: actions/upload-artifact@ee69f02b3dfdecd58bb31b4d133da38ba6fe3700
         with:
           name: image-digest ${{ matrix.name }}
-          path: image-digest
+          path: image-digest/${{ matrix.name }}.txt
```

Comment:
Can you clarify in the commit message what is wrong with the current path?

Reply:
Of course. You actually did well to ask me that because looking over it again: I think I made a mistake in my initial analysis.

I thought the upload artifacts was using the wrong path, which caused [the warning here](https://user-images.githubusercontent.com/4659919/119577198-733ee300-bdba-11eb-8c8e-f093e7c69434.png), but now I actually think the root cause is not the artifacts path but the fact we don't build artifacts for `build-and-push-with-qemu` for PR events, only for push events.

I pushed a more proper fix -- PTAL.

Reply:
For context following offline discussion:

- `build-and-push-with-qemu` builds `cilium-test`, which is used by the `nightly.yaml` workflow.
- Nightly runs only a schedule, hence why the current setup in `images-legacy.yaml` only builds for pushes to `master`.
- In the past, there was a nightly Jenkins job that could be triggered by `test-nightly` on PRs. It has since been marked as obsolete, probably when we moved nightly tests from Jenkins to GitHub, and removing the ability to test PRs in the process: https://jenkins.cilium.io/view/Obsolete/job/Cilium-PR-Nightly/